src/gui/attributetable/qgsdualview.cpp

```diff
@@ -73,7 +73,7 @@ QgsDualView::QgsDualView( QWidget *parent )
   auto createShortcuts = [ = ]( const QString & objectName, void ( QgsFeatureListView::* slot )() )
   {
     QShortcut *sc = QgsGui::shortcutsManager()->shortcutByName( objectName );
-    Q_ASSERT( sc ); // the shortcut must have been registered in the shortcuts manager
+    //Q_ASSERT( sc ); // the shortcut must have been registered in the shortcuts manager
```

Comment:
Is it normal to comment this? What happen if shortcut hasn't been registered

Reply:
That's a left over.
The issue is that it was crashing when running the tests locally, no idea why it works on Travis.

Reply:
Maybe a mac issue?